apps/aecore/src/aec_chain_state.erl

```diff
@@ -1158,3 +1165,17 @@ calc_rewards(FraudStatus1, FraudStatus2, GenerationFees,
     LockedAmount = TotalBlockAmount - B1Amt - B2Amt,
     {B1Amt, B2Amt, LockedAmount}.
 
+reward_split_for_protocol_foundation(BeneficiaryReward1, BeneficiaryReward2, NewestNodeHeight) ->
+    ActivationHeight = aec_governance:protocol_beneficiary_activation(NewestNodeHeight),
+    ActivationFlag = aec_governance:protocol_beneficiary_enabled(),
+    if ActivationHeight and ActivationFlag ->
+        ContribFactor = aec_governance:protocol_beneficiary_factor(),
+        Contrib1 = BeneficiaryReward1 * ContribFactor div 1000,
+        AdjustedBeneficiaryReward1 = BeneficiaryReward1 - Contrib1,
+        Contrib2 = BeneficiaryReward2 * ContribFactor div 1000,
```

Comment:
Explicating:
This is portion of the sum of the following items (or zero if miner of new key block was subsequently reported to be fraudulent):
* 60% of the tx fees (i.e. explicit tx fees plus gas fees) in generations started by old key block.
* Coinbase of new key block.
* Fraud reward of the old key block (i.e. 5% of the coinbase of the old key block) if the miner of the old key block was subsequently reported to be fraudulent.